WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -495,7 +495,12 @@ def getInputData( self, jobID ):
     if not res['OK']:
       return res
 
-    return S_OK( [ i[0] for i in res['Value'] if i[0].strip() ] )
+    inputData = [ i[0] for i in res['Value'] if i[0].strip() ]
+    for index, lfn in enumerate( inputData ):
```

Comment:
Why not just
`return S_OK( [ i[0].strip().replace('LFN:','') for i in res['Value'] if i[0].strip() ] )`

Reply:
Do you expect` LFN:` to be spelled `lFn:` or `lfn:`?   ;-)

Reply:
If so, you can still do it simpler:
```
inputData = []
for lfn in [ i[0] for i in res['Value'] if i[0].strip() ]:
  if lfn.lower().startswith('lfn:'):
    lfn = lfn[4:]
  inputData.append(lfn)
```

Reply:
But I just create one list, so less storage overhead

Reply:
Maybe, but do you care?... unless you have millions of input files, it doesn't matter, right?
...and in addition the code looks pretty much like Fortran then ;-) 

If you want a single list:

```
return S_OK( [ i[0].strip().[4:] if i[0].strip().lower().startswith('lfn:') else i[0].strip() for i in res['Value'] if i[0].strip() ])
```

but this not really readable!

Reply:
Which is why I prefer my solution.